.travis.yml

```diff
@@ -3,17 +3,17 @@ sudo: false
 
 matrix:
   include:
-    - python: "3.4"
     - python: "3.5"
     - python: "3.6"
     - python: "3.7"
       dist: xenial
       sudo: true
 
 install:
-  - curl -L https://github.com/TankerHQ/dmenv/releases/download/v0.5.0/dmenv-linux -o dmenv
+  - curl -L https://github.com/TankerHQ/dmenv/releases/download/v0.14.0/dmenv-linux -o dmenv
```

Comment:
0.15 ?

Reply:
no need for 0.15 here, we never use `dmenv run` because we're already in the virtualenv created by travis :)